testsuite/forge-cli/src/main.rs

```diff
@@ -1966,8 +1966,8 @@ fn realistic_env_max_load_test(
             LatencyBreakdownThreshold::new_with_breach_pct(
                 vec![
                     (LatencyBreakdownSlice::QsBatchToPos, 0.35),
-                    // only reaches close to threshold during epoch change
-                    (LatencyBreakdownSlice::QsPosToProposal, 0.6),
+                    // quorum store backpressure is relaxed, so queueing happens here
+                    (LatencyBreakdownSlice::QsPosToProposal, 2.5),
```

Comment:
Should we set this threshold based on the expected TPS? If QsPosToProposal is 2.5 seconds for even 1000 TPS workload, it's something we should look into.

Reply:
Yup this is just for this one test. I will have to adjust this for the rest of the tests based on forge stable results